src/Playwright.NUnitTest/Playwright.NUnit.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Title>Microsoft.Playwright.NUnit</Title>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net48</TargetFrameworks>
+    <PackageId>Microsoft.Playwright.NUnit</PackageId>
+    <Summary>A set of helpers and fixtures to enable using Playwright in NUnit tests.</Summary>
+    <Description>
+      Playwright enables reliable end-to-end testing for modern web apps. This package brings in additional helpers
+      and fixtures to enable using it within NUnit.
+    </Description>
+    <DebugSymbols>true</DebugSymbols>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <RunWithWarnings>true</RunWithWarnings>
+    <RootNamespace>Microsoft.Playwright.NUnit</RootNamespace>
+    <ReleaseVersion>0.0.0</ReleaseVersion>
+    <BuildFromSource>True</BuildFromSource>
+    <AssemblyName>Microsoft.Playwright.NUnit</AssemblyName>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <IsPackable>true</IsPackable>
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+  </PropertyGroup>
+  <Import Project="../Common/Version.props" />
+  <Import Project="../Common/SignAssembly.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\Playwright\Playwright.csproj" />
```

Comment:
One think we should do is declaring this as a nuget dependency.

Reply:
That happens automatically apparently. Looking at the produced nuget package:

![image](https://user-images.githubusercontent.com/1932563/120514189-4fd4f300-c3cd-11eb-9dd2-a588b2abfe04.png)